packages/shared/src/authz/domain/authErrors.ts

```diff
@@ -45,16 +70,21 @@ export const WorkspaceSsoProviderNotFoundError = defineAuthError({
   message: 'The workspace SSO provider was not found'
 })
 
-export const WorkspaceSsoSessionInvalidError = defineAuthError({
-  code: 'WorkspaceSsoSessionInvalid',
-  message: 'Your workspace SSO session is invalid'
-})
-
 export const WorkspaceSsoSessionNotFoundError = defineAuthError({
   code: 'WorkspaceSsoSessionNotFound',
   message: 'Your workspace SSO session was not found'
 })
 
+export const WorkspaceSsoSessionNoAccessError = defineAuthError<
+  'WorkspaceSsoSessionNoAccess',
+  {
+    workspaceSlug: string
```

Comment:
We need to pass along the workspaceSlug cause this error should cause a redirect to the workspace in FE